lib/offset.rb

```diff
@@ -0,0 +1,27 @@
+require 'date'
+
+class Offset
+  attr_reader :date
+
+  def initialize(date = Date.today.strftime("%d%m%y"))
```

Comment:
This could probably be organized where I set this to a method on current date and then use this snippet of code in that method to call on.